docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1399,6 +1375,7 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/audit/serviceProperties/geoCity`|Dallas|
 |`fhirServer/audit/serviceProperties/geoState`|TX|
 |`fhirServer/audit/serviceProperties/geoCounty`|US|
+|`fhirServer/bulkdata/isExportPublic`|true|
```

Comment:
Should we add the other bulkdata properties here too?  Do any of them have a "default"?

Reply:
The others are either with fixed values, or bucket configuration or password or encryption key, so I didn't add them here.